### PR TITLE
Refactor ping utility code

### PR DESCRIPTION
**Pull request by lokeshbv**
- Add new configs to control ping utility in build system
- Fix coding rule violations
- Fix divide by zero problem in ping utility output

### CODE REVIEW

**Comment by sunghan-chang** [comment]
@lokeshbv Please check compilation errors .

**Comment by seinfra** [comment]
Target : [aaa705886c7cd3b9807e73f24c163b9f42e82b29] - Code Rule Check (C++) OK.

File: apps/system/utils/netcmd_ping.c [hunk 1]
```diff
@@ -313,9 +313,9 @@ int ping_process(int count, const char *taddr)
 	close(s);
 
 	printf("--- %s ping statistics ---\n", taddr);
-	printf("%d packets transmitted, %d received, %f\%% packet loss,\n", ping_send_counter,
+	printf("%d packets transmitted, %d received, %f\%% packet loss\n", ping_send_counter,
 		   g_ping_recv_counter,
-		   (100.0f * (float)(g_ping_recv_counter - ping_send_counter))/(float)g_ping_recv_counter);
+		   100.0f * (g_ping_recv_counter ? (float)(g_ping_recv_counter - ping_send_counter)/(float)g_ping_recv_counter : 1));
```
Comment (pillip8282) [inline apps/system/utils/netcmd_ping.c]: I think that calculating the loss rate is wrong. it should be divide by g_ping_send_count not g_ping_recv_count.

Comment (lokeshbv) [reply apps/system/utils/netcmd_ping.c]: yes, I will modify

**Comment by seinfra** [comment]
Target : [cbe46ed5c854e8fbc8975cdfa63a159d6c35e8c2] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [cbe46ed5c854e8fbc8975cdfa63a159d6c35e8c2] - Code Rule Check OK.

**Comment by lokeshbv** [comment]
@pillip8282 review comment addressed please check

**Pull request merged**